libr/main/rasm2.c

```diff
@@ -320,16 +320,14 @@ static int rasm_disasm(RAsmState *as, char *buf, ut64 offset, int len, int bits,
 		}
 		clen = len; // XXX
 		data = (ut8 *)buf;
-	} else if (ascii) {
```

Comment:
Mmm isn't this going to change the behaviour? Without this it will go through the `else` branch and it will do `r_hex_str2bin` which expects an hexstring, doesn't it?

Reply:
yes, it changes the behaviour. but this behaviour was only used for the brainfuck plugin for unnecessary reasons. because it works well without this and having an arch-speciric if in an assembler is a bad idea